wordpress-mobile/WordPress-Android
# Issue/12934 add header to p2 postcard to show author

Fixes #12934

This PR modifies a header of the P2 reader posts to show the author's avatar alongside the blog avatar and show the author's name alongside the blog's name.
Where the p2 has not blavatar only the author's avatar will be shown (or no avatar at all if neither are available).

| Before | After Dark | After Light |
|--------|--------|--------|
| [![Image from Gyazo](https://i.gyazo.com/56a9ffafa4d9717bf3e3ddf067a9c898.png)](https://gyazo.com/56a9ffafa4d9717bf3e3ddf067a9c898) |[![Image from Gyazo](https://i.gyazo.com/900e6c9adc83c6cabfcb90d3f9c91e8a.png)](https://gyazo.com/900e6c9adc83c6cabfcb90d3f9c91e8a)|[![Image from Gyazo](https://i.gyazo.com/f8e2694a58729421cb121badf7b5d55f.png)](https://gyazo.com/f8e2694a58729421cb121badf7b5d55f)|

To test:
- To test DB migration install this branch over any previous version (eg. develop). Older posts already stored in DB without P2 flag will not show P2 specific layout.
- Make sure you follow a p2 site.
- Navigate to the reader and check that the p2 site's postcard header show's author's avatar next to the blog avatar.
- Make sure that the author's name is shown next to the blog name.
- Make sure those changes are not visible on non-p2 posts or in post details (including details of P2 post).

PR submission checklist:
